mk/update-travis-yml.py

```diff
@@ -121,6 +121,11 @@ def format_entries():
             %(sources)s"""
 
 def format_entry(os, target, compiler, rust, mode):
+    if os == "linux" and compiler == "gcc-5" and rust == "nightly" and mode == "DEBUG":
```

Comment:
I picked nightly because passing `-C link-dead-code` to rustc using RUSTFLAGS currently only works on beta and nightly. I can change to stable and it will start working when the next release happens. Until then coverage tracing won't be as accurate.

I picked gcc-5 as an example. I wanted to run kcov on just one build. Should I do it on all of them?

The debug symbols are needed for coverage tracking. Should I change the release builds to add them too on this pr?